scripts/init_mac.sh

```diff
@@ -0,0 +1,12 @@
+# Pull Submodules
```

Comment:
Added a script for OSX to set up a local development environment for `xpring-js` using `brew` according to the `CONTRIBUTING.md`.

Reply:
this also seems like a useful thing to leave in if it worked for you... maybe I should add a note about this in CONTRIBUTING